test/pkg.jl

```diff
@@ -41,7 +41,6 @@ temp_pkg_dir() do
   Pkg.rm("Example")
   @test isempty(Pkg.installed())
   @test !isempty(Pkg.available("Example"))
-  @test_throws ErrorException Pkg.available("FakePackageDoesn'tExist")
```

Comment:
But the whole `Pkg` thing is totally botched with `cd` calls required for `git` commands. On the next iteration of the `Pkg` rewrite `cd` calls will be purged to make it work with absolute paths. And such tests will be easy to implement without directory changing shenanigans. See #12710.